arch/risc-v/src/mpfs/Kconfig

```diff
@@ -58,6 +58,66 @@ config MPFS_OPENSBI
         ---help---
                 This uses a ld-envm-opensbi.script linker script and the mpfs_opensbi.c code to use external OpenSBI.
 
+config MPFS_HART0_SBI
+	bool "HART0 boots via SBI"
+	depends on MPFS_OPENSBI
+	default n
+
+config MPFS_HART1_SBI
+	bool "HART1 boots via SBI"
+	depends on MPFS_OPENSBI
+	default n
+
+config MPFS_HART2_SBI
+	bool "HART2 boots via SBI"
+	depends on MPFS_OPENSBI
+	default n
+
+config MPFS_HART3_SBI
+	bool "HART3 boots via SBI"
+	depends on MPFS_OPENSBI
+	default n
+
+config MPFS_HART4_SBI
+	bool "HART4 boots via SBI"
+	depends on MPFS_OPENSBI
+	default n
+
+config MPFS_HART0_ENTRYPOINT
+	hex "Application entry point for HART0"
+	depends on MPFS_BOOTLOADER
+	default 0xFFFFFFFFFFFFFFFF
```

Comment:
Could you please use lower case? I see that upper case is used only in one file https://github.com/apache/incubator-nuttx/blob/master/boards/arm/cxd56xx/spresense/Kconfig and other use lower case. I will make a PR to change `spresense/Kconfig` to  lower case